trio/_dtls.py

```diff
@@ -518,21 +549,23 @@ def encode_volley(self, messages, mtu):
 COOKIE_LENGTH = 32
 
 
-def _current_cookie_tick():
+def _current_cookie_tick() -> int:
     return int(trio.current_time() / COOKIE_REFRESH_INTERVAL)
 
 
 # Simple deterministic and invertible serializer -- i.e., a useful tool for converting
 # structured data into something we can cryptographically sign.
-def _signable(*fields):
+def _signable(*fields: bytes) -> bytes:
     out = []
     for field in fields:
         out.append(struct.pack("!Q", len(field)))
         out.append(field)
     return b"".join(out)
 
 
-def _make_cookie(key, salt, tick, address, client_hello_bits):
+def _make_cookie(
+    key: bytes, salt: bytes, tick: int, address: Any, client_hello_bits: bytes
```

Comment:
Is `address` `Any` for convenience's sake?

Reply:
It stems from the return type of `_SocketType.recvfrom`. I'll add a comment to update it in whichever gets merged later.

Reply:
Ah, ok.